### DISCUSSION TITLE
How to write Math equation

### DISCUSSION THREAD

**Discussion by tienthanh214** [mmistakes/minimal-mistakes#3054] [Q&A] [answered]
I used $$equation$$ in markdown (https://rmarkdown.rstudio.com/authoring_basics.html)
But it just show \[equation\].
So how can i write a math equation in a post?

**Comment by iBug** [comment, marked as answer]
You need to add MathJax (or KaTeX if you bother) library to your site before math works.

See how I do with my site: <https://github.com/iBug/iBug-source/blob/65d9cd59ee580e4e043da4c08e5e65f5642fc1fc/_includes/head/custom.html#L7-L19>  
And an example post: <https://github.com/iBug/iBug-source/blob/master/_posts/2019/2019-09-25-convert-dfa-to-regex.md>

Note the use of `{% if page.mathjax %}` allows per-page toggling by specifying `mathjax: true` in the front matter, so MathJax will not be loaded on pages that don't need it.
